ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -167,6 +167,7 @@ class MessagesHomeScreen extends React.Component<Props, State> {
         searchType="Messages"
         appLogo={true}
       >
+        <NavigationEvents onWillFocus={this.onRefreshMessages} />
```

Comment:
The last commits remove the auto-refresh of both message list and service list so that, after the cache is cleared, the user has to manually refresh the content